recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ test:
     {% endfor %}
     - test -f ${PREFIX}/include/bzlib.h  # [unix]
     - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]
```

Comment:
Maybe we should test for the non-versioned library instead. Otherwise this will be brittle.